src/main/java/io/specto/hoverfly/junit/core/model/GlobalActions.java

```diff
@@ -13,12 +13,16 @@
 package io.specto.hoverfly.junit.core.model;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Simulation model is part of a public API, it would be good if user wants to uses it and it just work out of the box. 
Also it doesn't have huge benefit for making this change which we need to update all the instance of objectMappers in the tests and always remember to configure it or use a preconfigure objectmapper when we need to work with the simulation model.